test/acceptance/cli-test.js

```diff
@@ -319,40 +382,64 @@ describe('ember-template-lint executable', function() {
       });
 
       describe('given a directory with errors and a lintrc with rules', function() {
-        it('should print properly formatted error messages', function() {
-          let result = run(['.', '--config-path', '../with-errors/.template-lintrc'], {
-            cwd: './test/fixtures/without-errors',
+        it('should print properly formatted error messages', async function() {
+          setProjectConfigWithoutErrors();
+
+          let projectWithErrors = new Project();
+          projectWithErrors.chdir();
+          projectWithErrors.setConfig({
+            rules: {
+              'no-bare-strings': true,
+            },
+          });
+
+          let result = run(['.', '--config-path', projectWithErrors.path('.template-lintrc.js')], {
+            cwd: project.path('.'),
           });
 
           expect(result.exitCode).toEqual(1);
           expect(result.stdout.split('\n')).toEqual([
-            path.resolve('./test/fixtures/without-errors/app/templates/application.hbs'),
+            project.path('app/templates/application.hbs'),
             '  1:4  error  Non-translated string used  no-bare-strings',
-            '  2:5  error  Non-translated string used  no-bare-strings',
+            '  1:39  error  Non-translated string used  no-bare-strings',
             '',
             '✖ 2 problems (2 errors, 0 warnings)',
           ]);
           expect(result.stderr).toBeFalsy();
+          await projectWithErrors.dispose();
         });
       });
 
       describe('given a directory with errors but a lintrc without any rules', function() {
-        it('should exit without error and any console output', function() {
-          let result = run(['.', '--config-path', '../without-errors/.template-lintrc'], {
-            cwd: './test/fixtures/with-errors',
+        it('should exit without error and any console output', async function() {
+          setProjectConfigForErrors();
+
+          let withoutErrorsProject = new Project();
```

Comment:
Why not use the shared `project` from the top?

Reply:
same as above